app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -91,6 +92,17 @@ def localized_locations(locations)
         end
       end
 
+      def scrub_message(message)
```

Comment:
There's a `scrub_message` method here, in the enter_password_controller, and in get_usps_proofing_results_job.rb.  I recognize that's repetitive.  I considered pulling the `scrub_message` method up into the application_controller.  I'm curious about other ways to DRY this up.  Thoughts?

Reply:
Update: I created the `IppHelper`.